bin/dh_virtualenv

```diff
@@ -23,12 +23,9 @@ import logging
 import os
 import sys
 
-# The debpython resides here
-sys.path.insert(1, '/usr/share/python/')
-
-from debpython.debhelper import DebHelper
 from dh_virtualenv import Deployment
 from dh_virtualenv.cmdline import get_default_parser
+from dh_virtualenv.debhelper import DebHelper
```

Comment:
Btw, maybe I should have vendored this in earlier, the sys path trickery has always been... not so nice, and this is so much cleaner 👍